README.md

```diff
@@ -2,90 +2,300 @@
+generated solution will target 64-bit CPUs by default. To manually target specific 
+architectures, tell CMake the intended architecture using the `-A` flag, as shown below. 
+Supported arguments for using this flag with Babylon Native include `Win32` for 32-bit 
+processors and `x64` for 64-bit processors.
+
+```
+cmake -A Win32 ..
+```
+
+CMake will generate a new `BabylonNative.sln` file in your working directory. Please
+be patient; this process can take several minutes. When the process is completed,
+open `BabylonNative.sln` by double-clicking on it in Windows Explorer or by entering 
+the following command:
+
+```
+start BabylonNative.sln
+```
+
+By default, the "Playground" demo app should be set as the Visual Studio start-up
+project. Build and run this app by pressing the green "Play" button or by pressing
+`F5` on your keyboard.
+
+### **Building on Windows 10, Targeting the Universal Windows Platform (UWP)**
+
+**Required Tools:** [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) with 
+C++ and UWP development tools, [Python 3.0](https://www.python.org/) or newer (required 
+by dependencies)
+
+For Windows development, CMake will generate a Visual Studio Solution. By default it 
+will target Win32, so to target UWP (which CMake describes as "Windows Store") you have 
+to specify certain CMake variables. From a terminal with access to CMake located in 
+your `BabylonNative/Build` directory, type the following (amend paths for alternative 
+build directories):
+
+```
+cmake -D CMAKE_SYSTEM_NAME=WindowsStore -D CMAKE_SYSTEM_VERSION=10.0 ..
+```
+
+Note that, by default, this will target the same processor architecture that is being
+used to build the solution; so if your developer computer has 64-bit CPU, CMake's 
+generated solution will target 64-bit CPUs by default. To manually target specific 
+architectures, tell CMake the intended architecture using the `-A` flag, as shown below. 
+Supported arguments for using this flag with Babylon Native include `Win32` for 32-bit 
+processors, `x64` for 64-bit processors, `arm` for ARM processors, and `arm64` for 
+ARM64 processors.
+
+```
+cmake -D CMAKE_SYSTEM_NAME=WindowsStore -D CMAKE_SYSTEM_VERSION=10.0 -A arm64 ..
+```
+
+CMake will generate a new `BabylonNative.sln` file in your working directory. Please
+be patient; this process can take several minutes. When the process is completed,
+open `BabylonNative.sln` by double-clicking on it in Windows Explorer or by entering 
+the following command:
+
+```
+start BabylonNative.sln
+```
+
+By default, the "Playground" demo app should be set as the Visual Studio start-up
+project. Build and run this app by pressing the green "Play" button or by pressing
+`F5` on your keyboard.
+
+### **Building on macOS, Targeting macOS**
+
+**Required Tools:** [Xcode 11](https://developer.apple.com/xcode/) or newer, 
+[Python 3.0](https://www.python.org/) or newer (required by dependencies)
```

Comment:
Python is bundled by default on MacOS. For cmake, use brew.

Reply:
Can you suggest a change? For now, I will merge and we can update it after.